### PR TITLE
[OKTA-459221] Authenticators contextual landing page

### PR DESCRIPTION
**Pull request by danielmaharry-okta**
## Description:
- **What's changed?** Added new overview page for authenticators guide section
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-459221](https://oktainc.atlassian.net/browse/OKTA-459221)

View at https://62473a70c30f455fd325bcd4--reverent-murdock-829d24.netlify.app/docs/guides/authenticators-overview/main/

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md [hunk 1]
```diff
@@ -1,8 +1,36 @@
 ---
-title: Authenticators overview
+title: Authenticators Overview
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md:2]:
```suggestion
title: Authenticators overview
```

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md:2]: Our titles use sentence case, as per style guide: https://github.com/okta/okta-developer-docs/wiki/Style-Guide#capitalization

File: packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md [hunk 2]
```diff
@@ -1,8 +1,36 @@
...
 <ApiLifecycle access="Limited GA" /><br>
 
-As you build your authentication use cases in your app, Okta gives you the option to further protect user sign-ins through the use of authenticators.
+Increase the security of your app by requiring a user to verify their identity in more than one way.
+
+---
+
+## Overview
+
+Strengthen your application's sign-in process by adding multiple **authentication factors**, or ways for a user to confirm their identity. Factors are enabled in the Okta org by creating a **policy** with one or more authenticators, and then assigning that policy to your app. A policy manages both the types of authenticators and which ones are required or optional. **Multifactor Authentication** (MFA) means that the policy requires at least two factors for a user to sign in. For example, an application can require using Okta Verify in addition to a username and password.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md:14]: Use of bold. Only for UI elements, as per our style guide: https://github.com/okta/okta-developer-docs/wiki/Style-Guide#bold

Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md:14]: The sentence "Factors are enabled" is a little confusing to me ... just the use of factors, really.  Same with the first "Authentication factors" .... I think these are better termed "Authenticators".  All Authenticators have factor types, which is why the user of just factors might be confusing. See: https://help.okta.com/oie/en-us/Content/Topics/identity-engine/authenticators/configure-authenticators.htm

Just my thoughts on this. Best to get a SME to chime in.

File: packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md [hunk 3]
```diff
@@ -1,8 +1,36 @@
...
+
+* [Okta Email authenticator](/docs/guides/authenticators-okta-email/aspnet/main/)
+* [Okta Verify](/docs/guides/authenticators-okta-verify/aspnet/main/)
+* [Google Authenticator](/docs/guides/authenticators-google-authenticator/aspnet/main/)
+* [FIDO2 (WebAuthn) authenticators](/docs/guides/authenticators-web-authn/aspnet/main/), such as a USB security key, fingerprint, FaceID, or Windows Hello
+
+In each guide, you'll learn
+
+How to enable the authenticator in your Okta org.
+How to add the enrollment and challenge flows to your application for that authenticator
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md:34]:
```suggestion
How to add the enrollment and challenge flows to your application for that authenticator.
```

File: packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md [hunk 4]
```diff
@@ -1,8 +1,36 @@
...
+The following guides step through adding a type of authenticator to your app using the Embedded SDK for your platform:
+
+* [Okta Email authenticator](/docs/guides/authenticators-okta-email/aspnet/main/)
+* [Okta Verify](/docs/guides/authenticators-okta-verify/aspnet/main/)
+* [Google Authenticator](/docs/guides/authenticators-google-authenticator/aspnet/main/)
+* [FIDO2 (WebAuthn) authenticators](/docs/guides/authenticators-web-authn/aspnet/main/), such as a USB security key, fingerprint, FaceID, or Windows Hello
+
+In each guide, you'll learn
+
+How to enable the authenticator in your Okta org.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md:33]: Were these meant to be bullets?

**Review by brianduffield-okta — CHANGES_REQUESTED** [review]
A few change requests.

File: packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md (hunk 1 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md:2]: fixed

File: packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md (hunk 2 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md:14]: fixed

File: packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md (hunk 3 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md:34]: fixed

File: packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md (hunk 2 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md:14]: The wording was pre-approved, but I've added the link to that page and its parent page listing all the authenticators.

File: packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md (hunk 4 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md:33]: Yes it was. No idea why they aren't there.

**Pull request merged**